nordicsemi/dfu/package.py

```diff
@@ -111,6 +112,7 @@ class Package(object):
     DEFAULT_SD_ID = [0xFFFE]
     DEFAULT_DFU_VER = 0.5
     MANIFEST_FILENAME = "manifest.json"
+    DEFAULT_BOOT_VALIDATION_TYPE = ValidationTypes.VALIDATE_GENERATED_CRC.name
```

Comment:
Shouldn't default validation type be NO_VALIDATION? When default is CRC people that want to keep using the "old" functionality for nrfutil will have to update their calls to specify that no boot validation should be used.

Reply:
After a discussion with Øyvind we agreed that CRC as default is correct, and settings page will be updated to have CRC as default as well.